angular/projects/spark-angular/src/lib/components/sprk-autocomplete/sprk-autocomplete.stories.ts

```diff
@@ -0,0 +1,599 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkAutocompleteModule } from './sprk-autocomplete.module';
+import { SprkAutocompleteComponent } from './sprk-autocomplete.component';
+import { SprkInputModule } from '../../directives/inputs/sprk-input/sprk-input.module';
+import { SprkInputContainerModule } from '../inputs/sprk-input-container/sprk-input-container.module';
+import { SprkIconModule } from '../sprk-icon/sprk-icon.module';
+import { SprkIconComponent } from '../sprk-icon/sprk-icon.component';
+import { SprkLabelModule } from '../../directives/inputs/sprk-label/sprk-label.module';
+import { SprkLabelDirective } from '../../directives/inputs/sprk-label/sprk-label.directive';
+import { SprkIconInputContainerModule } from '../inputs/sprk-icon-input-container/sprk-icon-input-container.module';
+import { SprkInputContainerComponent } from '../inputs/sprk-input-container/sprk-input-container.component';
+import { SprkFieldErrorModule } from '../../directives/inputs/sprk-field-error/sprk-field-error.module';
```

Comment:
If all of the needed directives and components are importing and exported in the autocomplete module file then they all do not also need to imported here. This would mimic what an app would encounter. They should only need to import the autocomplete file. You will have to import the ones that are needed in the `subcomponents` property on the default story object so that we can display their documentation. But shouldnt have to import things like the `SprkInputModule`, `SprkIconComponent` etc. As those modules should be imported/exported in the autocomplete module as it uses them.